debug_toolbar/templates/debug_toolbar/redirect.html

```diff
@@ -4,8 +4,12 @@
   <head>
     <title>Django Debug Toolbar Redirects Panel: {{ status_line }}</title>
     <script{% if toolbar.csp_nonce %} nonce="{{ toolbar.csp_nonce }}"{% endif %} type="module" src="{% static 'debug_toolbar/js/redirect.js' %}" async></script>
+    <link{% if toolbar.csp_nonce %} nonce="{{ toolbar.csp_nonce }}"{% endif %} rel="stylesheet" href="{% static 'debug_toolbar/css/toolbar.css' %}">
   </head>
-  <body>
+  <body id="djDebug" {% if toolbar.store_id %}data-store-id="{{ toolbar.store_id }}"{% endif %}>
+    <div class="djdt-deprecation-warning">
+      <strong>{% translate "WARNING:" %}</strong> {% translate "The RedirectsPanel is deprecated and will be removed in a future version. The HistoryPanel now provides the ability to view toolbar data for redirected requests. If you still have a use case for this panel, please comment on this <a href='https://github.com/django-commons/django-debug-toolbar/issues/2216'>issue</a>" %}
```

Comment:
```suggestion
      <strong>{% translate "WARNING:" %}</strong> {% translate "The RedirectsPanel is deprecated and will be removed in a future version. The HistoryPanel now provides the ability to view toolbar data for redirected requests. If you still have a use case for this panel, please comment on this <a target='_blank' href='https://github.com/django-commons/django-debug-toolbar/issues/2216'>issue</a>" %}
```

I also think a `{% blocktranslate %}` may make this a bit more readable since you could add more linebreaks to the code.